questions-answers-application/welcome.vb

```diff
@@ -14,4 +14,21 @@ Public Class welcome
         LoadQuestions()
     End Sub
 
+    Private Sub CreateQuestionToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CreateQuestionToolStripMenuItem.Click
+        createQuestion.ShowDialog()
+        LoadQuestions()
+    End Sub
+
+    Private Sub UpdateQuestionToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles UpdateQuestionToolStripMenuItem.Click
+        Dim updatequestionform As New updateQuestion(getQuestionID())
+        updatequestionform.ShowDialog()
+    End Sub
+
+    Public Function getQuestionID() As Integer
+        Return getquestionvalue("id")
+    End Function
+
+    Public Function getquestionvalue(ByVal column As String)
+        Return dgvQuestions.Item("id", dgvQuestions.CurrentRow.Index).Value
```

Comment:
This method is broken because you have hard-coded the column name `"id"` here, so it will only ever return the ID, and no other column value. See [here][my-getqv] how I pass the `column` variable through to the `dgv.Item` method which will allow us to dynamically access any column value.

[my-getqv]:https://github.com/MyTeamName/questions-answers-app/blob/14e4df438ec644fe6e5c70e15c53a49d468bdd0d/questions-answers-application/welcome.vb#L33